src/components/Rankings.js

```diff
@@ -19,10 +21,37 @@ function Rankings({ coverageData }) {
     }
   }
 
+  const SORT_FUNCTIONS = {
+    Ascending: (a, b) => a.percentage - b.percentage,
+    Descending: (a, b) => b.percentage - a.percentage,
+    Alphabetical: (a, b) => a.name.localeCompare(b.name),
+    ReverseAlphabetical: (a, b) => b.name.localeCompare(a.name),
+  };
+
+  function changeSort(event) {
+    setFields([...fields].sort(SORT_FUNCTIONS[event.target.value]));
+  }
+
   return (
     <div className="flex-auto">
       <div className="bg-white p-2 rounded-widgit">
-        <h1 className="font-bold text-xl pb-3">Rankings</h1>
+        <div className="flex flex-row justify-between">
+          <h1 className="font-bold text-xl pb-3">Rankings</h1>
+          <select id="sortSelect" onChange={changeSort}>
+            <option value="Ascending" key="Ascending">
```

Comment:
It might be nice if this was linked to the SortFunctions object itself, instead of being loosely linked. Don't feel too strongly about this since it's such a small feature, but something to consider/describe why you opted not to.